backend/src/controllers/claimController.ts

```diff
@@ -137,11 +138,24 @@ export const getClaims = async (req: Request, res: Response) => {
     if (status) {
       statusFilter = { status };
     }
+
+    // add filter for memberAddress if provided
+    let memberFilter: Prisma.ClaimWhereInput = {};
+    if (memberAddress) {
+      memberFilter = {
+        wage: {
+          userAddress: memberAddress,
+          teamId: teamId,
+        },
+      };
+    }
+
```

Comment:
This is not optimal. 
You just need the memberAddress, so there is no need to add the team Id, It's like how you are doing the status filter.